pkg/rpc/client/wsclient.go

```diff
@@ -287,7 +318,7 @@ func (c *WSClient) performSubscription(params request.RawParams) (string, error)
 	var resp string
 
 	if err := c.performRequest("subscribe", params, &resp); err != nil {
-		return "", err
+		return "", fmt.Errorf("failed to subscribe: %w", err)
```

Comment:
Do we need this additional data? It's always returned from some `Subscribe*` method, so it's obvious that we have some kind of subscription failure.